ndscan/experiment/parameters.py

```diff
@@ -275,6 +283,39 @@ def changed_after_use(self) -> bool:
         """
         return self._changed_after_use
 
+    @host_only
+    def _get_toplevel_handle(self) -> "ParamHandle":
+        """
+        Get the highest level ParamHandle in the chain of bound parameters
+
+        Walks the DAG of bound parameters to find the highest level handle.
+        That may be this ParamHandle if this parameter is not rebound.
+        """
+        if self._parent_handle is None:
+            return self
+        else:
+            return self._parent_handle._get_toplevel_handle()
+
+    @host_only
+    def _get_all_handles_for_param(self) -> list["ParamHandle"]:
```

Comment:
Nitpick: The name of this seems slightly misleading, as it does not actually return all the handles for the parameter this points to. `_get_all_{bound,child}_handles()`?